docs/Overlays.md

```diff
@@ -31,6 +31,7 @@ Currently, the following overlay vars are provided out of the box by Distillery:
   - release_version: Version of the release being built
   - is_upgrade: Is this release an upgrade release
   - upgrade_from: The version of the release being upgraded from, nil if not an upgrade.
+  - soft_purge: Is this appup using soft purge when upgrade
```

Comment:
Let's change this a bit to "Should appups use soft purge during upgrade (see appup docs for more info)"